ui/components/component-library/avatar-favicon/README.mdx

```diff
@@ -14,44 +14,36 @@ The `AvatarFavicon` is an image component that renders an icon that is provided
 
 ## Props
 
-The `AvatarFavicon` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+The `AvatarFavicon` accepts all props below as well as all [Box](/docs/components-componentlibrary-box--docs#props) component props
 
 <ArgsTable of={AvatarFavicon} />
 
-`AvatarFavicon` accepts all [AvatarBase](components-componentlibrary-avatarbase--default-story))
-component props
-
-<ArgsTable of={AvatarBase} />
-
 ### Size
 
-Use the `size` prop to set the size of the `AvatarFavicon`.
-
-Optional: `AVATAR_FAVICON_SIZES` from `./ui/components/component-library` object can be used instead of `Size`
+Use the `Size` prop or the `AvatarFaviconSize` enum from `../../component-library` to change the size of `AvatarFavicon`
```

Comment:
Update sentence to refer to the `size` prop and enum
```suggestion
Use the `size` prop and the `AvatarFaviconSize` enum from `../../component-library` to change the size of `AvatarFavicon`
```

Reply:
updated [`here`](https://github.com/MetaMask/metamask-extension/pull/20430/commits/6774ca92db19fd39d3c0a46652d3a410c89eff0d)